docs/developer/developing.md

```diff
@@ -0,0 +1,49 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in different terminal windows 
```

Comment:
Maybe:

> Run the following in parallel (for example, in different terminals).

Reply:
continued in https://github.com/openshift/cincinnati/pull/198#discussion_r375427729